ember-app/app/models/health.js

```diff
@@ -0,0 +1,53 @@
+import Settings from 'app/models/settings';
+import ajax from 'ic-ajax';
+import Ember from 'ember';
+var get = Ember.get;
+
+
+var Health = Settings.extend({
+  repo: {},
+  storageKey: Ember.computed('repo.data.repo.full_name',{
```

Comment:
Cool! For troubleshooting convenience, we should may add a refresh button that refetches all the checks ?